apps/test/unit/templates/progress/ProgressLessonTest.js

```diff
@@ -151,30 +151,44 @@ describe('ProgressLesson', () => {
     assert.equal(wrapper.find('FontAwesome').at(0).props().icon, 'caret-down');
   });
 
-  it('starts collapsed if it is not the current stage', () => {
+  it('starts collapsed for student if it is not the current stage', () => {
     const wrapper = shallow(
       <ProgressLesson
         {...defaultProps}
+        viewAs={ViewType.Student}
         currentStageId={2}
       />
     );
     assert.equal(wrapper.state('collapsed'), true);
   });
 
-  it('starts uncollapsed if it is the current stage', () => {
+  it('starts uncollapsed for teacher even if not the current stage', () => {
```

Comment:
There's  hole in your test matrix here:

&nbsp; | Is current stage | Not current stage |
--- |:---:|:---:
**Teacher** | |  ✓ |
**Student** |  ✓ |  ✓ |